src/instance/commands/triggers/calculate.ts

```diff
@@ -28,6 +28,9 @@ export default class Calculate extends ChatCommandHandler {
       .replaceAll(',', '') // removes commas from numbers
 
     const result = evalExpression(expression)
+    if (result <= 50 && Math.random() < 0.2) {
+      return `${context.username} haiyaaaaaaaaa this is so easy, you're a disappointement *takes off slipper* (answer is ${result.toLocaleString()})`
```

Comment:
```suggestion
      return `${context.username} haiyaaaaaaaaa this is so easy, you're a disappointement *takes off slipper* (answer: ${result.toLocaleString()})`
```
Change output from `answer >>> is <<< $ANSWER` to `answer: $ANSWER` to match other output.